web-console/future-openshift/operators-overview/operators-overview.md

```diff
@@ -0,0 +1,18 @@
+# Showing Operators on Overview
+
+This documentation details how operators will be surfaced on the Status page. Rather than exposing CSV objects, the status page should now expose the CRs being managed by identified CSVs in designated groups.
+
+## Group by: Application
+![image](img/operators8-app.png)
+* In the group by Application view, operator managed CRs will be grouped together with the CSV object name as the group header (AMQ Streams).
+* In this example, the “View operator details” link will bring users to the Installed Operators > AMQ Streams > All Instances tab. The “View resources” link will bring users to the Installed Operators > AMQ Streams > Kafka details > Resources tab.
+
+## Group by: Resource
+![image](img/operators8-resource.png)
+* In the group by Resource view, CRs will be grouped with other CRs of the same type, with the API type as the group header.
+
+## Side panel
+![image](img/operators8-overview.png)
```

Comment:
Shouldn't we want to expose the `Resources` tab on the side panel too? (similar to `Installed Operators > AMQ Streams > Kafka details > Resources tab`)

Reply:
@tlwu2013 Based on feedback from the OLM team at the sprint demo, we decided to not surface the resources in the side panel. The reasoning is that users shouldn't care too much about the resources that make up an operator, but if it's having problems we still allow them to see that if they drill down. The overall thought being that ideally, the user would not be interested in the things that make up the operator, because that's what the operator should care about rather than the user. Let me know your thoughts on this!

Reply:
The resources page (Installed Operators > AMQ Streams > Kafka details > Resources tab) is also easily accessible from the "View resources" link, so viewing those resources is still pretty accessible!

Reply:
@tlwu2013 We originally had the resources tab here but Daniel did not want us repeating that page and preferred we link over to the object details where the user can view that same information.